packages/web/src/store/storeContext.ts

```diff
@@ -77,6 +78,7 @@ export const buildStoreContext = ({
   instagramRedirectUrl: env.INSTAGRAM_REDIRECT_URL,
   share: getShare(isMobile),
   audiusSdk,
+  reinitializeSdk: initSdk,
```

Comment:
nit, maybe this should just be called initSdk? like, there's nothing different about initializing vs. reinitializing, but the reader may think this does something different?